katello.gemspec

```diff
@@ -49,6 +49,7 @@ Gem::Specification.new do |gem|
 
   # Pulp
   gem.add_dependency "anemone"
```

Comment:
Can we drop anemone and webrick deps now?

Reply:
Removed anemone.. 🎉